source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -288,30 +325,31 @@ The SDAM specification defines `when
 <https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#connection-pool-creation>`_
 the driver should create connection pools.
 
-Once a pool is created, if minPoolSize is set, the pool MUST immediately
-begin creating enough connections such that totalConnections >= minPoolSize.
-These connections MUST be created in a non-blocking manner. The driver SHOULD
-set up (connect, handshake and authenticate) these connections in a non-blocking
-manner, such as via the use of a background thread or asynchronous I/O.
+Once a pool is created, if minPoolSize is set, the pool MUST
+immediately begin populating enough `Connections <#connection>`_ such that
+totalConnections >= minPoolSize. These `Connections <#connection>`_ MUST be
```

Comment:
I think the intention is to populate such that the number of available + in use connections is at least min pool size. Simply creating dummy connection objects doesn't achieve the goal of min pool size of having that many *ready* connections.